lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Server.hs

```diff
@@ -131,6 +131,51 @@ spec = do
                 logged `shouldNotContain` "TRACE"
                 logged `shouldNotContain` "DEBUG"
 
+        it "LOGGING - Serve debug logs for one component" $ \ctx -> do
+            withTempFile $ \logs hLogs -> do
+                let cmd = Command
+                        (commandName @t)
+                        ["serve"
+                        , "--node-port", show (ctx ^. typed @(Port "node"))
+                        , "--random-port"
+                        , "--genesis-block-hash", block0H
+                        , "--log-level", "debug"
+                        , "--trace-stake-pool-db", "debug"
+                        ]
+                        (pure ())
+                        (UseHandle hLogs)
+                void $ withBackendProcess nullTracer cmd $ do
+                    threadDelay (5 * oneSecond)
+                hClose hLogs
+                logged <- T.lines <$> TIO.readFile logs
+                let grep str = filter (T.isInfixOf str)
+                let allDebugLogs = grep "Debug" logged
+                let componentDebugLogs = grep "stake-pool-db" allDebugLogs
+                let networkDebugLogs = grep "network" allDebugLogs
+                length componentDebugLogs `shouldNotBe` 0
+                length networkDebugLogs `shouldBe` 0
```

Comment:
so we here are serving with log-level=debug. after 5 seconds we close log handle. and then look into logs "greping" Debug ones. And we do not see neither stake-pool-db nor network related logs. What I do not undestand is why this test is titled `Serve debug logs for one component` Don't we want to show that something is present here, rather than absent?

Reply:
ah, my bad. there is `shouldNotBe` - yes, that was what I expected to capture!!!! :+1: